resolwe/flow/migrations/0001_squashed_0043_full_text_search.py

```diff
@@ -27,6 +32,23 @@ class Migration(migrations.Migration):
     operations = [
         CITextExtension(),
         TrigramExtension(),
+        UnaccentExtension(),
+        migrations.RunSQL(
```

Comment:
Why is this configuration duplicated? In general, you should leave existing migrations as is.

Reply:
I have removed it from the base migration.